xmlschema is not available on Solaris.

test_pdb_continue_with_recursive_debug breaks for yet unknown reasons and
should be investigated more (though it's not a core functionality we are
using Pytest for).

Two tests need to be skipped because cwd is not removable on Solaris.

--- pytest-8.3.3/testing/test_junitxml.py
+++ pytest-8.3.3/testing/test_junitxml.py
@@ -10,8 +10,6 @@ from typing import cast
 from typing import TYPE_CHECKING
 from xml.dom import minidom
 
-import xmlschema
-
 from _pytest.config import Config
 from _pytest.junitxml import bin_xml_escape
 from _pytest.junitxml import LogXML
@@ -22,6 +20,7 @@ from _pytest.reports import BaseReport
 from _pytest.reports import TestReport
 from _pytest.stash import Stash
 import pytest
+pytest.skip("Xmlschema is not available", allow_module_level=True)
 
 
 @pytest.fixture(scope="session")
--- pytest-8.3.3/testing/test_debugging.py
+++ pytest-8.3.3/testing/test_debugging.py
@@ -667,6 +667,7 @@ class TestPDB:
         child.expect("1 passed")
         self.flush(child)
 
+    @pytest.mark.skip(reason="doesn't work on Solaris")
     @pytest.mark.parametrize("capture_arg", ("", "-s", "-p no:capture"))
     def test_pdb_continue_with_recursive_debug(
         self, capture_arg, pytester: Pytester
--- pytest-8.3.3/testing/_py/test_local.py
+++ pytest-8.3.3/testing/_py/test_local.py
@@ -621,6 +621,7 @@ class TestLocalPath(CommonFSTests):
         assert str(local()) == os.getcwd()
 
     @skiponwin32
+    @pytest.mark.skip("cwd cannot be removed on Solaris")
     def test_chdir_gone(self, path1):
         p = path1.ensure("dir_to_be_removed", dir=1)
         p.chdir()
@@ -634,6 +635,7 @@ class TestLocalPath(CommonFSTests):
                 raise NotImplementedError
 
     @skiponwin32
+    @pytest.mark.skip("cwd cannot be removed on Solaris")
     def test_chdir_gone_in_as_cwd(self, path1):
         p = path1.ensure("dir_to_be_removed", dir=1)
         p.chdir()
